cloudpassage/csp_findings.py

```diff
@@ -0,0 +1,61 @@
+"""CspFinding class"""
+
+from .halo_endpoint import HaloEndpoint
+import cloudpassage.sanity as sanity
+
+
+class CspFinding(HaloEndpoint):
+    """Initializing the CspAccount class:
+
+    Args:
+        session (:class:`cloudpassage.HaloSession`): This will define how you
+            interact with the Halo API, including proxy settings and API keys
+            used for authentication.
+
+    Keyword args:
+        endpoint_version (int): Endpoint version override.
+
+    """
+
+    object_name = "csp_finding"
+    objects_name = "csp_findings"
+    default_endpoint_version = 1
+
+    def endpoint(self):
+        """Return endpoint for API requests."""
+        return "/v{}/{}".format(self.endpoint_version, self.objects_name)
+
+    def pagination_key(self):
+        """Return the pagination key for parsing paged results."""
+        return self.objects_name
+
+    def object_key(self):
+        """Return the object key for parsing detailed results."""
+        return self.object_name
+
+    def describe(self, rule_id):
```

Comment:
Let's remove describe here (raise NotImplementedError) because this will end up behaving differently from other abstractions with `describe`. Instead of this, let's just document `list_all()` well, with filtering options.